task2.py

```diff
@@ -0,0 +1,13 @@
+#Multi-paradigm programming language: Task 1, Maslova Vitaliia, idk
+
+print('Multi-paradigm programming language: Task 2')
+
+print('Maslova Vitaliia, idk')
```

Comment:
unnecessary blank line
```suggestion
print('Multi-paradigm programming language: Task 2')
print('Maslova Vitaliia, idk')
```